core/network/src/protocol/consensus_gossip.rs

```diff
@@ -627,6 +629,13 @@ mod tests {
 		}
 	}
 
+	macro_rules! stream_assert_eq {
+		($stream: expr, $expected_value: expr) => {
+			let mut rt = Runtime::new().unwrap(); // use tokio 0.1's runtime on stable rust
```

Comment:
Before we just used `poll` why can we not use `poll_next` now?

Reply:
Do you mean `.await`? You can only use it in async functions but test function cannot be async

Reply:
Just use [this](https://docs.rs/futures-preview/0.3.0-alpha.17/futures/executor/fn.block_on_stream.html), then you don't need to import tokio.